luminaire/utils/validation.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+import numbers
+
+def check_random_state(seed):
+    """Turn seed into a np.random.RandomState instance
+    Parameters
+    ----------
+    seed : None, int or instance of RandomState
+        If seed is None, return the RandomState singleton used by np.random.
+        If seed is an int, return a new RandomState instance seeded with seed.
+        If seed is already a RandomState instance, return it.
+        Otherwise raise ValueError.
+    """
+    if seed is None or seed is np.random:
```

Comment:
It would be good to put this whole thing in a try/except block. Also, raising an error and failing the code is the right way here?